app/controllers/proceeding_types_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProceedingTypesController < ApplicationController
+  def index
+    proceeding_types = ProceedingType.all
+    render json: ProceedingTypesSerializer.new(proceeding_types).serialized_json, status: :ok
```

Comment:
**[note/not a blocker]** More for spreading the knowledge than being required as a change, by default render json already returns a successful response status code unless we tell it otherwise, so `status: :ok` I believe can be optional in the cases we're returning a successful response.